um_canonicalize/tags.yaml

```diff
@@ -151,6 +176,7 @@ categories:
     - MASC
     - FEM
     - NEUT
+    - MASC+FEM
```

Comment:
The standard name for this in traditional grammar is "uter".

I just took a look at Dutch adjectives in Unimorph 2. Dutch has an uter/neuter distinction in indefinite adjective inflection (for instance) but it looks like the uters are just not marked with gender and the neuters are marked with NEUT.

Reply:
Uters should be marked in Dutch. That being said, I'm not convinced that `MASC+FEM` is correct for the three languages where it's explicitly used. Better would be `non{NEUT}`.